src/App.js

```diff
@@ -35,13 +28,13 @@ function App() {
         <Route path='/' element={<Home/>} />
         <Route path='/create' element={<Create/>} />
         <Route path='*' element={<NotFound />}/>
+        <Route path="/edit/:id" element={<Create/>}/>
         <Route path= "/creators/:id" element={<BlogDetail/>}/>
         </Routes>
 
       
         <Navbar/> 
```

Comment:
The Navbar component should be above all the Routes. So it should be on line 26

Reply:
I have done this sir